IDEOorg/steps
# #161468059 show completed for client profile only

#### What does this PR do?
- Add implementation to ensure that the numbers of completed tasks only display when viewing the profile of a client. This should not be seen when the profile to view is for an admin or a coach.

#### Description of Task to be completed?
- In `apps/admin/src/layouts/UserLayout.tsx` check for the type of user. Display the text only if user.type === 'Client'.

#### How should this be manually tested?
- Clone the repo and `cd` to steps. Run `npm install` and step your environment using the sample in .example.env. Start the app by run `NODE_ENV=development npm run start` You should be able to view the app at `http://localhost:3000/signup`. Create an admin account and verify that the profile displayed does not include the text `Task completed: 0`.

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
#161468059

#### Screenshots (if appropriate)
##### Formal behaviour
![image](https://user-images.githubusercontent.com/30448913/47557097-c5b87a00-d907-11e8-9988-01a1d44e16db.png)

##### Present behaviour 
![image](https://user-images.githubusercontent.com/30448913/47556991-8f7afa80-d907-11e8-87a1-e5cc7e5b2993.png)

##### View client profile
![image](https://user-images.githubusercontent.com/30448913/47557231-10d28d00-d908-11e8-8183-d8fb575c7f7c.png)
